src/App.js

```diff
@@ -1,7 +1,31 @@
-import "./App.css";
+import React, { Component } from 'react'
+import productData from "./data/productData"
+import ProductsList from "./Components/ProductsList"
+import Checkout from './Components/Checkout'
+import Cart from "./Components/Cart"
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+export class App extends Component {
+  constructor(){
+    super()
+    
+    this.state = {
+      cartItems: productData
+    }
+  }
 
-export default App;
+  handleClick = () => {
+    
+  }
+
+  render() {
+    return (
+      <div>
+        <ProductsList productData={productData} handleClick={this.handleClick}/>
+        <Checkout />
+        <Cart displayCart={this.state.cartItems} />
```

Comment:
The Checkout and Cart components both need to know the subtotal, tax, and total price of the shopping cart. I'd recommend computing those values here inside the render method of App and passing those values down to each component as props.